ipn/ipnlocal/local.go

```diff
@@ -2269,6 +2283,17 @@ func (b *LocalBackend) WatchNotifications(ctx context.Context, mask ipn.NotifyWa
 		if mask&ipn.NotifyInitialNetMap != 0 {
 			ini.NetMap = b.netMap
 		}
+		if mask&ipn.NotifyInitialTailfsShares != 0 && b.tailfsSharingEnabledLocked() {
+			shares, err := b.tailfsGetSharesLocked()
+			if err != nil {
+				b.logf("unable to notify initial tailfs shares: %v", err)
```

Comment:
This will only happen if they're unable to read their state store, or it contains corrupted Tailfs share information. If the state store is unreadable or corrupted, I assume all sort of stuff is going to go haywire?